webdriver-spec.html

```diff
@@ -5316,106 +5316,110 @@
-
-    <p>The diagram below displays when each action gets executed.
-     "Source 1" is the first finger, and "source 2" is the second.
-
-    <p><img src=graphics/note1actions.svg alt="">
-
-    <p>There is no limit to the number of input sources,
-     and there is no restriction regarding the length of each input's action list.
-     Meaning, there is no requirement that all action lists have to be the same length.
-     It is possible for one input source's action list
-     may have more actions than another.
-
-     In this case, the action list for the first finger contains 2 actions
-     (<code><a>pointerDown</a></code>, <code><a>pointerUp</a></code>),
-     and the action list for the second finger contains 3
-     (<code><a>pointerDown</a></code>, <code><a>pointerMove</a></code>, <code><a>pointerUp</a></code>).
-
-     <p>And the execution of each action will be done as follows:
-
-     <p><img src=graphics/note4actions.svg alt="">
-
-     <p>Specific timing for the actions can also be expressed.
-      The <code><a>pause</a></code> action can be used to either a) indicate
-      a specific amount of time an input source must wait, or b)
-      can be used to signify that the current input source must wait
-      until all other actions in the tick are completed.
-      For the former case, the current tick being executed must wait
-      for the longest pause to complete.
-      For example, in this diagram:
-
-     <p><img src=graphics/note2actions.svg alt="">
-
-     <p>The remote end will dispatch the <code><a>pointerDown</a></code> action in the first tick.
-      In the second tick, since source 1 declares a <code><a>pause</a></code> of 5 seconds,
-      the remote end will dispatch the <code><a>pointerUp</a></code> event for source 2,
-      and will wait 5 seconds before moving on to executing the third tick.
-
-     <p>In the event that one tick contains multiple <code><a>pause</a></code> durations,
-      the remote end will wait the maximum duration
-      before moving on to executing the next tick.
-
-     <p>As noted before, <code><a>pause</a></code> can be used to signify inaction during a tick.
-      If <code><a>pause</a></code> is declared without a time period,
-      then the input source will not have any actions executed in the containing tick.
-      As an example:
-
-     <p><img src=graphics/note3actions.svg alt="">
-
-     <p>During tick 2, source 1 will have its pointerMove action dispatched,
-      while source 2 will do nothing.
-  </aside>
-
-  <p>
-
-  </section> <!-- /Introduction -->
+<section>
+<h2>Actions</h2>
+
+<p class=issue>Expand these examples
+
+<p>The Actions API provides a low-level interface
+ for providing virtualised device input to the web browser.
+ Conecputally, the actions API divides time into a series of <a>ticks</a>.
+ The local end sends a series of actions
+ which correspond to the change in state, if any,
+ of each input device during each tick.
+ For example, pressing a key
```

Comment:
By [semantic linefeeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/), which I find particularly useful for maintaining prose in VCSes. I don’t expect you to do the same.